src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml.cs

```diff
@@ -57,10 +69,17 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
                 && lvm.InitializeCommand != null)
             {
                 ViewModel = null;
-                lvm.InitializeCommand.Execute(null);
 
                 _ = Task.Run(async () =>
                 {
+                    // You know, this creates the situation where we wait for
+                    // both loading page properties, AND the items, before we
+                    // display anything.
+                    //
+                    // We almost need to do an async await on initialize, then
+                    // just a fire-and-forget on FetchItems.
+                    lvm.InitializeCommand.Execute(null);
```

Comment:
Yeah, this was my note elsewhere about if we could use the async enumerable pattern or something else here, it's definitely something we'll want to tweak.

Reply:
You know, after I added the manual calls to update the `Name`/`Loading` in the original call to `InitializeProperties` (in c168e59), all of a sudden this isn't a problem anymore. 

Maybe I can just can #187 entirely